e2e/src/app.e2e-spec.ts

```diff
@@ -1,23 +1,43 @@
 import { AppPage } from './app.po';
-import { browser, logging } from 'protractor';
+import { browser, by, element } from 'protractor';
 
 describe('workspace-project App', () => {
   let page: AppPage;
+  let modalDialog;
 
   beforeEach(() => {
     page = new AppPage();
+    modalDialog = page.getModalDialog();
   });
 
-  it('should display welcome message', () => {
+  it('should display table with 8 action items', () => {
     page.navigateTo();
-    expect(page.getTitleText()).toEqual('examp-ng-app app is running!');
+    expect(page.getActionItems().count()).toBe(8);
   });
 
-  afterEach(async () => {
-    // Assert that there are no errors emitted from the browser
-    const logs = await browser.manage().logs().get(logging.Type.BROWSER);
-    expect(logs).not.toContain(jasmine.objectContaining({
-      level: logging.Level.SEVERE,
-    } as logging.Entry));
+  it('should open and close dialog modal', () => {
+    page.navigateTo();
+    page.getOpenDialogButton().click();
+    browser.wait(modalDialog.isDisplayed, 5000);
+    page.getCloseDialogButton().click();
+    expect(page.getActionItemsTable()).toBeTruthy();
+  });
+
+  it('should open a modal dialog, fill inputs, create action item and closed dialog', () => {
+    page.getOpenDialogButton().click();
+    browser.wait(modalDialog.isDisplayed, 3000);
```

Comment:
We can move these 2 lines to page object:
```
page.getOpenDialogButton().click();
browser.wait(modalDialog.isDisplayed, 5000);
```

and call `openDialog()` function in tests.